great-idea/product.html

```diff
@@ -0,0 +1,138 @@
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+
+<div class="item">
+  <h2>UX / UI</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+
+<div class="item">
+<h2>Digital Marketing</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+<div class="item">
+    <h2>Web Development</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+<div class="item">
+    <h2>iOS Development</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+<div class="item">
+<h2>Android Development</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+<div class = "line"></div>
+
+<div class = "menu-bottom">
+<h2>Some Facts About Our Products</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+</div>
+
+
+<ul>
+<li>Awesome thing</li>
+<br>
+<li>Amazing thing</li>
+<br>
+<li>Cool thing</li>
+<br>
+<li>Great thing</li>
+<br>
+</ul>

```

Comment:
It may be cleaner to add spacing between these list items with `line-height` in the css properties.